packages/@lightningjs/ui-components/src/components/Control/Control.mdx

```diff
@@ -80,7 +80,7 @@ Optional flag for the `Control` element to collapse (hide the title) when `Contr
 
 ### Style Properties
 
-`Control` contains the same style properties as [Button](?path=/docs/controls-button--button) in additional to a couple other style properties listed below:
+`Control` contains the same style properties as [Button](?path=/docs/components-button--button) in additional to a couple other style properties listed below:
```

Comment:
Path for the button works fine in the style properties, but is not working in properties section, also noticed they are few other spots that needs to be updated like in TitleRow, Control and few other component docs.
I can submit another quick PR to update those.

Reply:
I have created a ticket for this, may be after merging this PR and Mandy's PR. That should be ready to be picked. https://ccp.sys.comcast.net/browse/LUI-939